cpp/src/plasma/store.cc

```diff
@@ -802,6 +802,33 @@ Status PlasmaStore::ProcessMessage(Client* client) {
         WarnIfSigpipe(send_fd(client->fd, object.store_fd), client->fd);
       }
     } break;
+    case fb::MessageType::PlasmaCreateAndSealRequest: {
+      std::string data;
+      std::string metadata;
+      unsigned char digest[kDigestSize];
+      RETURN_NOT_OK(ReadCreateAndSealRequest(input, input_size, &object_id, &data,
+                                             &metadata, &digest[0]));
+      int device_num = 0;
```

Comment:
This should be passed through as part of ReadCreateAndSealRequest

Reply:
I added a comment explaining that it is not supported yet.